examples/print_layer/vklayerprint.py

```diff
@@ -99,27 +100,31 @@ def referenced_types(commands: list):
     'layer_prefix': 'Printer',
 }
 
-tmp = env.from_string(open('examples/templates/layerprint.cpp.jinja2').read())
+generated_dir = '{}/generated'.format(currentdir)
```

Comment:
Just FYI, because there are a lot of places here using `format`. You can also use the following, which I think is a bit shorter and easier to read:

```
generated_dir = f'{currentdir}/generate'
```

Reply:
Done.